services/management/backup/backups_service.go

```diff
@@ -83,14 +112,45 @@ func (s *BackupsService) StartBackup(ctx context.Context, req *backupv1beta1.Sta
 		RetryInterval: req.RetryInterval.AsDuration(),
 	})
 	if err != nil {
-		return nil, err
+		return nil, convertBackupError(err)
 	}
 
 	return &backupv1beta1.StartBackupResponse{
 		ArtifactId: artifactID,
 	}, nil
 }
 
+func convertRestoreBackupError(restoreError error) error {
```

Comment:
Can't we merge this func and `convertBackupError` into one?

Reply:
In my opinion these 2 convert functions are different, it is better to not have just one convert function that covers all the cases, because we will not be able to
 - cover in the tests the cases that is not used in the specific function
 - know what errors the function could return